apiserver/facades/client/spaces/shims.go

```diff
@@ -41,33 +22,6 @@ func NewStateShim(st *state.State) (*stateShim, error) {
 	}, nil
 }
 
-// AllMachines returns all machines and maps it to a corresponding common type.
-func (s *stateShim) AllMachines() ([]Machine, error) {
-	allStateMachines, err := s.State.AllMachines()
-	if err != nil {
-		return nil, err
-	}
-	all := make([]Machine, len(allStateMachines))
-	for i, m := range allStateMachines {
-		all[i] = &machineShim{m}
-	}
-	return all, nil
-}
-
-// AllConstraints returns all constraints in the model,
-// wrapped in the Constraints indirection.
-func (s *stateShim) AllConstraints() ([]Constraints, error) {
-	found, err := s.State.AllConstraints()
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	cons := make([]Constraints, len(found))
-	for i, v := range found {
-		cons[i] = v
-	}
-	return cons, nil
-}
-
 func (s *stateShim) ConstraintsBySpaceName(spaceName string) ([]Constraints, error) {
```

Comment:
This should be removed as well. I haven't checked, but if it's used then we need to replace it with the constraint domain.